### PR TITLE
Fix logic of computeClosestIJKAxis

### PR DESCRIPTION
**Pull request by bruyeret**
Two mappers with two different slicing modes could have the same ijkMode
For example, two mappers with X and Y slicing mode could both show slice J
Also remove the warning about inaccurate slicing because oblique slicing should be done with the new ImageResliceMapper and the slicing modes XYZ are intentionally used to snap to IJK

Add a function to vtkMath (with tests and docs)

### CODE REVIEW

File: Sources/Common/Core/Math/test/testMath.js [hunk 1]
```diff
@@ -754,3 +754,45 @@ test('Test singularValueDecomposition', (t) => {
...
+      matrix: [1, 0, 0, 0, 0, 1, 0, 1, 0],
+      expected: [0, 2, 1],
+    },
+    {
+      matrix: [1, 0, 0, 0, 0, -1, 0, 1, 0],
+      expected: [0, 2, 1],
+    },
+    {
+      matrix: [1, 0, 0, 0, 0, -1, 0, 1, 0],
+      expected: [0, 2, 1],
```
Comment (finetjul) [inline Sources/Common/Core/Math/test/testMath.js:777]: why twice the same test

File: Sources/Common/Core/Math/test/testMath.js [hunk 2]
```diff
@@ -754,3 +754,45 @@ test('Test singularValueDecomposition', (t) => {
...
+    },
+    {
+      matrix: [1, 0, 0, 0, 1, 0, 0, 0, 1],
+      expected: [0, 1, 2],
+    },
+    {
+      matrix: [1, 0, 0, 0, 1, 0, 0, 0, 1],
```
Comment (finetjul) [inline Sources/Common/Core/Math/test/testMath.js:784]: why twice the same test ?

File: Sources/Common/Core/Math/index.js [hunk 3]
```diff
@@ -2146,6 +2146,51 @@ export function getMajorAxisIndex(vector) {
   return axisIndex;
 }
 
+// This function iteratively associate a column with a row by choosing
+// the greatest absolute value from the remaining row and columns
+// Returns an array L such as if the matrix is column major,
+// the row i is associated with column L[i]
+export function getRowColumnAssociations(matrix, n = 3) {
```
Comment (finetjul) [inline Sources/Common/Core/Math/index.js:2153]: The documentation is not clear to me. 
It seems that you return an order for columns. What is this order ?

Comment (bruyeret) [reply Sources/Common/Core/Math/index.js:2153]: The order of the column indices is the order of the rows, this is what I mean by `the row i is associated with column L[i]`
How would you phrase this differently?

**Comment by bruyeret** [comment]
I renamed `getRowColumnAssociations` to `getSparseOrthogonalMatrix` and it now returns an orthogonal matrix of 0, 1 and -1
What do you think @finetjul

File: Sources/Common/Core/Math/test/testMath.js [hunk 4]
```diff
@@ -754,3 +754,47 @@ test('Test singularValueDecomposition', (t) => {
...
+
+test.only('Test getSparseOrthogonalMatrix', (t) => {
+  const successMsg = 'Row/column association is correct';
+  const tests = [
+    {
+      matrix: [
+        0.702, 0.7025, -0.1163, -0.2856, 0.1281, -0.9497, -0.6523, 0.7, 0.2906,
+      ],
+      expected: [0, 1, 0, 0, 0, -1, -1, 0, 0],
+    },
```
Comment (finetjul) [inline Sources/Common/Core/Math/test/testMath.js:766]: can you maybe also add as a test the problem you faced with Kitware Glance ?

**Comment by finetjul** [comment]
LGTM

File: Sources/Common/Core/Math/test/testMath.js (hunk 4 above)
Comment (bruyeret) [reply Sources/Common/Core/Math/test/testMath.js:766]: It is the first test matrix, which use to associate both X and Z to J

**Review by finetjul — APPROVED** [review]
LGTM

**Pull request merged**